README.md

```diff
@@ -81,6 +81,44 @@ end
 
 Can be `http` or `https` protocol
 
+## Add your own service
+
+You can make available your own services:
+
+`my_awesome_service.rb`:
+
+```ruby
+module MarkdownVideos::Services
+
+  class MyAwsomeService < ServiceBase
+
+    def self.regexp
+      /(https?:\/\/myaweso.me\/video\/([\w\-]{4,16})\??([\w\-\=]+)?)/
+    end
+
+    def width
+      560
+    end
+
+    def height
+      315
+    end
+
+    def url
+      "https://player.myaweso.me/video/#{id}"
```

Comment:
"probably", and I'm not saying that they wont be, I'm saying that there's no reason for it to be in position 2 in the regexp. Basically, when writing my service, I'd rather being explicit about how extract the required info from the url matching, rather than relying on the fact that you decided that the id is generally the second thing matched. It's fine if you leave it as is, just my opinion :)